config/gradle/publish.gradle

```diff
@@ -56,7 +74,7 @@ publishing {
     }
 }
 
-def isMaster(gitBranch) {
+static def isMaster(gitBranch) {
     return gitBranch != null && gitBranch != "" && gitBranch.equals("master");
```

Comment:
```suggestion
    return gitBranch != null && gitBranch.equals("master");
```

Reply:
this is cheating, you can't suggest changes to lines that aren't changed in the diff :laughing: